### PR TITLE
Update frhelper from 4.2.9 to 4.3.0

### PR DESCRIPTION
**Pull request by miccal**
Created by `brew bump`

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by miccal** [comment]
```
Version '4.3.0,1082' differs from '4.2.9,1081' retrieved by livecheck.
```
Not for me:
```
/opt/homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromDefaultTapPathLoader): loading frhelper

Cask:             frhelper
Livecheckable?:   Yes

URL:              https://www.eudic.net/update/frhelper_mac.xml
Strategy:         Sparkle

Matched Versions:
4.3.0,1082

frhelper: 4.2.9,1081 ==> 4.3.0,1082
```
I have been seeing this update for a while now: https://github.com/Homebrew/homebrew-cask/pull/142716

**Comment by reitermarkus** [comment]
For me it's `4.2.9,1081` locally.

File: Casks/frhelper.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "frhelper" do
-  version "4.2.9,1081"
-  sha256 "5eae043981f58139f734caa86404bb524cc57ff621b35d5ab15375d7ee6a06dc"
+  version "4.3.0,1082"
+  sha256 "fc5b9336231c41a44197389dcff32c9ebcffc1a5f6f902b711f79dda173c42c9"
 
   url "https://static.frdic.com/pkg/fhmac.dmg?v=#{version.csv.second}",
```
Comment (reitermarkus) [inline Casks/frhelper.rb:5]: The version here also doesn't seem to be the build number when downloading manually from the website.

I am getting `https://static.frdic.com/pkg/fhmac.dmg?v=2023-02-13` instead.

**Comment by reitermarkus** [comment]
The download does contain 4.3.0 for me though.

File: Casks/frhelper.rb (hunk 1 above)
Comment (miccal) [reply Casks/frhelper.rb:5]: Strange, I am getting `https://static.frdic.com/pkg/fhmac.dmg?v=1082` ... ?

Comment (reitermarkus) [reply Casks/frhelper.rb:5]: Weird.

**Comment by reitermarkus** [comment]
The `v` parameter in the URL doesn't seem to have an effect. I cannot download 4.2.9 with the old build number.

**Review by reitermarkus — DISMISSED** [review]

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**